### DISCUSSION TITLE
Are the files compressed for downloading? how are the diffs calculated?

### DISCUSSION THREAD

**Discussion by alonsovidales** [abraunegg/onedrive#2743] [Q&A] [answered]
We are planing to use the client to sync files in schools in Kenya and Tanzania that have very limited internet and they have to pay per MB, so, is this client compressing the files individually? how are the diffs being calculated?

Any ideas to reduce the network usage as much as possible?

Thank you!

**Comment by abraunegg** [comment, marked as answer]
@alonsovidales 
> is this client compressing the files individually? 

The Microsoft Graph API does not support *any* compression. The file size online and/or local is what will be transferred.

> Any ideas to reduce the network usage as much as possible?

If you are *not* using the files online, compress the files *first* then upload that compressed data to Microsoft OneDrive is your *only* solution.

Download and decompress the data post sync.
